libs/d3d12/main.c

```diff
@@ -206,3 +206,15 @@ HRESULT WINAPI DLLEXPORT D3D12GetInterface(REFCLSID rcslid, REFIID iid, void **d
         return E_NOINTERFACE;
     return IVKD3DCoreInterface_GetInterface(core, rcslid, iid, debug);
 }
+
+HRESULT WINAPI DLLEXPORT D3D12PIXEventsReplaceBlock()
```

Comment:
Does it really take no arguments? () != (void) in C fwiw. Proper return types is important for WINAPI calling convention.